blockmanager.go

```diff
@@ -267,8 +268,8 @@ type PeerNotifier interface {
 	TransactionConfirmed(tx *dcrutil.Tx)
 }
 
-// blockManangerConfig is a configuration struct for a blockManager.
-type blockManagerConfig struct {
+// BlockManagerConfig is a configuration struct for a blockManager.
+type BlockManagerConfig struct {
```

Comment:
This definitely should not be exported.  I plan to split it out into a separate package as we did upstream some time ago.  It'll end up as `newpkg.Config`.